metadata/blinky.ukp.informatik.tu-darmstadt.de%2Fshibboleth.xml

```diff
@@ -29,13 +32,13 @@
         <md:Extensions>
             <mdui:UIInfo xmlns:mdui="urn:oasis:names:tc:SAML:metadata:ui">
                 <mdui:DisplayName xml:lang="en">INCEpTION - Community Server</mdui:DisplayName>
-                <mdui:Description xml:lang="en">INCEpTION provides a semantic annotation platform offering intelligent annotation assistance and knowledge management.</mdui:Description>
+                <mdui:Description xml:lang="en">A semantic annotation platform offering intelligent annotation assistance and knowledge management.</mdui:Description>
                 <mdui:DisplayName xml:lang="de">INCEpTION - Community Server</mdui:DisplayName>
-                <mdui:Description xml:lang="de">INCEpTION ist eine semantische Annotationsplattform mit intelligenter Annotationshilfe und Wissensmanagement.</mdui:Description>
+                <mdui:Description xml:lang="de">Eine semantische Annotationsplattform mit intelligenter Annotationshilfe und Wissensmanagement.</mdui:Description>
                 <mdui:InformationURL xml:lang="en">https://inception-project.github.io/</mdui:InformationURL>
                 <mdui:Logo height="239" width="239">https://inception-project.github.io//assets/img/logo.png</mdui:Logo>
                 <mdui:Keywords xml:lang="en">INCEpTION annotation</mdui:Keywords>
-                <mdui:PrivacyStatementURL xml:lang="de">https://www.tu-darmstadt.de/datenschutzerklaerung.en.jsp</mdui:PrivacyStatementURL>
```

Comment:
If you want you can keep both the German and English versions as long as they are compatible.

Reply:
They are different, one is from our University and one specially crafted for this SP.